api/apps/api/src/modules/clone/export/application/export-project.handler.spec.ts

```diff
@@ -124,6 +124,11 @@ const getFixtures = async () => {
         },
       });
     },
+    ThenExportRequestsEventIsPresent(exportId: ExportId) {
+      expect(events[1]).toMatchObject({
```

Comment:
very minor - should we give a name to `events[1]` to make it clearer (to our or other future selves) what we're looking for in the array?

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/686/commits/21df305ec895e6d0a752b5ef266d08c4076a3094